integrated_channels/integrated_channel/management/commands/transmit_courseware_data.py

```diff
@@ -78,4 +78,14 @@ def send_data_task(username, channel_code, channel_pk):
         channel,
     )
 
-    channel.transmit_course_data(user)
+    # channel.transmit_course_data(user)
+    try:
+        channel.transmit_course_data(user)
+    except Exception:  # pylint: disable=broad-except
```

Comment:
We should really narrow down what the common errors are here, no?

Reply:
Actually, We did not know what exactly the exception occurring here.

Reply:
Yep I assumed so, but it would be nice to know what it is.

Reply:
i think we should catch this general exception, but we need to add logging information about this exception below. then we can start adding specific cases as we find them.